ui/DesignSystem/Component/SearchModal.svelte

```diff
@@ -0,0 +1,155 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import { push } from "svelte-spa-router";
+
+  import { Icon, Input } from "../Primitive";
+
+  import TrackToggle from "./TrackToggle.svelte";
+  import Urn from "./Urn.svelte";
+
+  import * as path from "../../src/path";
+  import { Status } from "../../src/remote";
+  import { updateUrn, validation } from "../../src/search";
+  import { ValidationStatus } from "../../src/validation";
+
+  let searchBar, value;
+
+  const navigateToProject = () => {
+    if ($validation.status !== ValidationStatus.Success) return;
+
+    dispatch("hide");
+    push(path.projectUntracked(value));
+  };
+
+  const onKeydown = ev => {
+    switch (ev.key) {
+      case "Escape":
+        dispatch("hide");
+        break;
+      case "Enter":
+        navigateToProject();
+        break;
+    }
+  };
+
+  const dispatch = createEventDispatcher();
+
+  const clickOutside = ev => {
+    if (ev.target !== searchBar && !searchBar.contains(ev.target)) {
+      showTrackingInfo = false;
+      dispatch("hide");
+    }
+  };
+
+  $: if (value && value.length > 0) {
+    updateUrn({ urn: value });
+  }
+
+  // TODO(sos): animate & show/hide based on actual remote response
+  $: showTrackingInfo =
+    value && value.length > 0 && $validation.status === Status.Success;
+</script>
+
+<style>
+  .overlay {
+    /* TODO(sos): ask brandonhaslegs for color & opacity for both light & dark modes */
+    background-color: black;
+    opacity: 0.7;
+    width: 100%;
+    height: 100%;
+    position: absolute;
```

Comment:
When pulling in the latest version of master, `position: absolute` won't work anymore. Use `position: fixed` instead.